core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -210,6 +212,8 @@
     private Disposable positionEventTimer;
     private PlaybackServiceNotificationBuilder notificationBuilder;
 
+    FeedMedia autoSkippedFeedMedia;
```

Comment:
Can this be private? I think it would be enough to only save the ID instead of the whole object.